src/FirmwarePlugin/APM/APMSensorParams.qml

```diff
@@ -92,4 +92,7 @@ Item {
     property bool compass2Calibrated:               compass2Available ? compass2OfsXFact.value != 0.0  && compass2OfsYFact.value != 0.0  &&compass2OfsZFact.value != 0.0 : false
     property bool compass3Calibrated:               compass3Available ? compass3OfsXFact.value != 0.0  && compass3OfsYFact.value != 0.0  &&compass3OfsZFact.value != 0.0 : false
     property var  rgCompassCalibrated:              [ compass1Calibrated, compass2Calibrated, compass3Calibrated ]
+
+    property Fact autoDeclinationFact:              factPanelController.getParameterFact(-1, "COMPASS_AUTODEC")
```

Comment:
Are these parameters available in all versions of ArduPilot firmware going all the way back to the min supported ArduPilot version for each vehicle type. If not, referencing them like this will cause the parameters missing error to pop depending on the firmware you are using.

Reply:
This parameter is over 9 years old. It works fine on latest releases, but I couldn't find the "min supported ardupilot version for each vehicle type" information anywhere...

I'll test the oldest versions we have metadata for, to be sure

Reply:
> but I couldn't find the "min supported ardupilot version for each vehicle type" information anywhere...

Forgot that I removed that code this release.

> This parameter is over 9 years old

This is fine then.